### PR TITLE
Refactor/change/drfat/0

### PR DESCRIPTION
**Pull request by Izobov**
Please `click on` the `(^) Preview` tab and select the appropriate sub-template:

- [Pull-Request Template into branch:dev](?expand=1&template=pr_to_dev.md)
- [Pull-Request Template into branch:main](?expand=1&template=pr_to_main.md)

### CODE REVIEW

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -541,6 +535,7 @@
 
 <svelte:window
   on:resize={() => {
+    sessionStore.updateData([["windowWidth", document.documentElement.clientWidth]]);
     $sessionStore.windowWidth = document.documentElement.clientWidth;
```
Comment (migbash) [inline src/routes/+layout.svelte:539]: The line: 539 is unecessary, as it's now handled by the line before it.

File: src/lib/components/misc/modal/ModalMain.svelte
Comment (migbash) [inline src/lib/components/misc/modal/ModalMain.svelte:1]: Missing the `boilerplate` comments for this file, those in the `boilerplate` file and component (script, html) separation comments, like the CSS one.

File: src/routes/+layout.svelte (hunk 1 above)
Comment (Izobov) [reply src/routes/+layout.svelte:539]: oh, yes,I just forget to remove it

File: src/lib/components/misc/modal/ModalMain.svelte
Comment (Izobov) [reply src/lib/components/misc/modal/ModalMain.svelte:1]: done

**Comment by Izobov** [comment]
@migbash check again

**Comment by migbash** [comment]
@Izobov Select a `Milestone` for the `PR`!

**Pull request merged**